documentation/modules/overview/con-security-configuration-fips.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-security-overview.adoc
+
+[id="security-configuration-fips_{context}"]
+= Federal Information Processing Standards (FIPS)
+
+[role="_abstract"]
+Federal Information Processing Standards (FIPS) are a set of security standards established by the US government to ensure the confidentiality, integrity, and availability of sensitive data and information that is processed or transmitted by information systems. 
+Strimzi supports FIPS compliance by automatically enabling FIPS mode on the OpenJDK used in Strimzi container images when running on a FIPS-enabled Kubernetes cluster.
+
+FIPS compliance requires the use of specific cryptographic algorithms and security protocols, which are supported in a FIPS-enabled Kubernetes cluster.
```

Comment:
I would probably avoid this as it over simplifies the relation between FIPS enabled Kubernetes and the algos. It is mostly just a flag which is carried over. I would remove the whole sentence or at least the second part I guess.